### ISSUE TITLE
Container scheduled tasks get added to SQL database but not executed by pteroq

### ISSUE THREAD

**Issue by bobthewomtad** [pterodactyl/panel#4338]
### Current Behavior

Creating scheduled tasks for backups or automated commands are added to MySQL/MariaDB but not executed by pteroq. This only applies to container tasks and not work for the panel such as adding users or notifying server installs that are reported via email. All scheduled tasks report as never being executed no matter how frequently they are scheduled. No errors are generated on a failure of scheduled tasking. Tasks will execute when selecting "Run Now", and debug messaging from Wings shows tasks being sent to the Panel cron.

### Expected Behavior

Scheduled tasks execute as prescribed in the scheduler, or if not executed, an error or notification of the failure of the scheduled task,

### Steps to Reproduce

Installed on Debian 11. Using the following

Using Maria DB
CACHE_DRIVER=memcached
SESSION_DRIVER=database
QUEUE_DRIVER=database

Verified panel resolves and not port conflicts. Eggs load from containers. Servers created are accessible. Verify pteroq cronworker is created and configured correctly.  Create any scheduled task, will not execute automatically, but will execute when selecting "Run Now" from the schedule menu.

### Panel Version

1.10.1

### Wings Version

1.7.0

### Games and/or Eggs Affected

Minecraft (Vanilla), Minecraft (Forge), Barotrauma, Source Engine (Garry's Mod)

### Error Logs

```bash
Wings https://ptero.co/ehapaxukyh  -no errors, but you can see the events being sent to the panel cron
Panel https://pteropaste.com/c4guq8 - cURL error due to panel access while restarting wings.
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by playaz44** [comment]
This is a known bug, on discord I received information that it has to do with it 
https://discord.com/channels/122900397965705216/936744899787784282/1009445961883455559

```
It's not a database issue, it's a request validation issue. Wings can't always be certain who the user/actor is for activities which is why they can be null. The panel is supposed to accept null values for actor and IP but the validation rule doesn't allow null values. See:
https://github.com/pterodactyl/panel/issues/4290
https://github.com/pterodactyl/panel/blob/develop/app/Http/Requests/Api/Remote/ActivityEventRequest.php
```

**Comment by bobthewomtad** [comment]
> This is a known bug, on discord I received information that it has to do with it https://discord.com/channels/122900397965705216/936744899787784282/1009445961883455559
> 
> ```
> It's not a database issue, it's a request validation issue. Wings can't always be certain who the user/actor is for activities which is why they can be null. The panel is supposed to accept null values for actor and IP but the validation rule doesn't allow null values. See:
> https://github.com/pterodactyl/panel/issues/4290
> https://github.com/pterodactyl/panel/blob/develop/app/Http/Requests/Api/Remote/ActivityEventRequest.php
> ```

This is not the same thing. Its not a database error with null user/actor information. Its the queue worker not executing all tasks from the DB. Panel user administration is fine, but any scheduled task is not done.

**Comment by danny6167** [comment]
If they are stuck "processing" then pteroq probably crashed and needs a restart. If they never attempt to run at all, then you probably have an issue with your cron setup. Either way, not a bug. Please use discord for support.

**Comment by bobthewomtad** [comment]
> If they are stuck "processing" then pteroq probably crashed and needs a restart. If they never attempt to run at all, then you probably have an issue with your cron setup. Either way, not a bug. Please use discord for support.

I have been attempting to use the discord for support for over a week. I have a second person also looking at the configuration of the cronworker and has verified its setting in accordance with the setup guide (https://pterodactyl.io/panel/1.0/getting_started.html#queue-listeners), and verified it is running normally. Wings reports execution of sftp transfers of cron tasks to the panel. Its not that the queue worker isn't functioning, its that it doesn't execute all the tasks in the database.

**Comment by danny6167** [comment]
the sftp messages have nothing to do with scheduled tasks. Wings doesn't "sftp transfer" cron tasks to the panel. That's not how any of this works.

**Comment by bobthewomtad** [comment]
TL:DR The ask is for some form of error to generate to assist with diagnosing tasks failing to execute.

Regardless of how it works, the ask is that there be an error generated when tasks are being scheduled and not accomplished. No matter how many time I upload logs for the bot to read, nothing provided any useful information outside of checking pteroq or the cron settings.

I have been able to restore task execution by swapping to redis for all drivers. Outside of reenabling redis in the pteroq file I made no other changes.

If you do not have documentation on how this works to provide someone in this implementation, you should be prepared to provide a resource on how it it is supposed to work or allow someone else to provide actionable assistance. That produces useful information where I as a user might ask more useful questions and provide more useful feedback instead of failing to address the issue.

**Comment by DaneEveritt** [comment]
Failed jobs are logged into `failed_jobs` with the error that was triggered. But that error _should_ be logged to the disk as well. If nothing is running then there is some configuration error with the queue worker. This issue is reported monthly, and it is always just a configuration issue.

You can also manually execute the command that should be triggered by the `pteroq` service and confirm it runs without errors being output.
